src/main/scala/lockbook/dev/UiOrchestrator.scala

```diff
@@ -114,6 +116,20 @@ class UiOrchestrator(
           repositoryUi.pullAllRepos()
         }
       })
+
+    val saveKeyCombo = new KeyCodeCombination(KeyCode.L, KeyCombination.META_DOWN)
+
+    stage
+      .sceneProperty() //unsure where I should add a listener to
+      .addListener((_, _, newv) => {
+        newv.addEventHandler(
```

Comment:
I'm concerned that `newv` could be `null`, see what the difference between this and [EditorUi is](https://github.com/lockbook/lockbook-dev-desktop/blob/master/src/main/scala/lockbook/dev/EditorUi.scala#L52)